### ISSUE TITLE
Cannot access Dashboard when locale is set to "en": HandleExceptions

### ISSUE THREAD

**Issue by ebeauchamps** [statamic/cms#1108]
I put back ```'locale' => 'en'``` in app.php (originally: fr or it)
The default site is displaying OK
After clicking on Log in to access the Dashboard, I get this error:

```
Facade\Ignition\Exceptions\ViewException
htmlspecialchars() expects parameter 1 to be string, array given (View: /Users/efb/Sites/statamic3/vendor/statamic/cms/resources/views/partials/nav-main.blade.php)
```

```
Illuminate\Foundation\Bootstrap\HandleExceptions::handleError
:251
vendor/laravel/framework/src/Illuminate/Support/helpers.php:251
```

The Dashboard is displaying OK when locale is "fr" or "it"

Any idea? Thanks in advance

**Comment by jasonvarga** [comment]
Please share your ignition error.

**Comment by ebeauchamps** [comment]
https://flareapp.io/share/LPdBJd5O?owner_token=GIGJHVq8QmvV88EegAdCKwdrXJqE3K0m#F77

**Comment by ebeauchamps** [comment]
Well.
So I created a brand new install

I started with ```en```then I tried playing with the different locales, visiting the CP in another locale and going back to English and visiting the CP again.

I eventually added back my latest French and Italian translations and did the same.

I never encountered again the same error.

I also played with various locale preferences in my user.yaml file, letting English as the primary locale in app.php

I finally added back one blueprint and some content from another install, and everything went OK, too.

So I guess we could close this issue, probably related to a glitch induced by playing with the previous install (I am using MAMP currently, and experiencing some CSRF token error one in a while, so I guess my version isn't that stable).

Is it oK with you?
